exoctk/exoctk_app/app_exoctk.py

```diff
@@ -437,7 +469,7 @@ def contam_visibility():
 
             # Make plot
             title = form.targname.data or ', '.join([form.ra.data, form.dec.data])
-            pG, pB, dates, vis_plot, table = vpa.using_gtvt(str(form.ra.data), str(form.dec.data), form.inst.data)
+            pG, pB, dates, vis_plot, table = vpa.using_gtvt(str(form.ra.data), str(form.dec.data), form.inst.data.split(' ')[0])
```

Comment:
@mfixstsci I'm having some PR de ja vu with this call to `split` but just for safety's sake, can you tell me why we need it/why it wasn't already there? I thought that was already merged in.

Reply:
yeah, this edit was from the most recent PR you approved of mine (https://github.com/ExoCTK/exoctk/pull/274)

Reply:
@jaymedina No wonder it's familiar. Why is this appearing as a new change here then?

Reply:
originally I thought maybe it's because @mfixstsci created this branch before I merged that PR in. but after thinking about it some more, he wouldn't have this new line at all if that's the case. so I'm not sure 🤔 either way, the line looks fine. we can test out `contam overlap` on tlexo again after this gets merged just to be safe.

Reply:
in a PR, does GitHub compare `master` branch to `edited branch` or does it compare `unedited branch` to `edited branch`? I just checked and this line change is already implemented on `master`, so idk why it shows up as a change here

Reply:
@jaymedina Thanks for looking into this. If the change looks right in both places I say we let it ride and just be sure to test it.

Reply:
@julesfowler I agree, and no problem